po/nb.po

```diff
@@ -21010,11 +21001,11 @@ msgstr "Vis priskolonnen"
 
 #. Translators: This is a ngettext(3) message, %d is the number of prices added
 #: gnucash/import-export/csv-imp/assistant-csv-price-import.cpp:1881
-#, fuzzy, c-format
+#, c-format
 msgid "%d added price"
 msgid_plural "%d added prices"
-msgstr[0] "Legg til en ny pris."
-msgstr[1] "Legg til en ny pris."
+msgstr[0] "% tillagt pris"
+msgstr[1] "%s tillagte priser"
```

Comment:
`%d`

Other errors, which I got, but you did not neccesary have caused them: msgfmt -c --statistics --check-accelerators="_" nb.po
nb.po:3520: msgstr lacks the keyboard accelerator mark '_'
nb.po:3633: msgstr lacks the keyboard accelerator mark '_'
nb.po:3914: msgstr lacks the keyboard accelerator mark '_'
nb.po:3992: msgstr lacks the keyboard accelerator mark '_'
nb.po:4093: msgstr lacks the keyboard accelerator mark '_'
nb.po:4405: msgstr lacks the keyboard accelerator mark '_'
nb.po:4413: msgstr lacks the keyboard accelerator mark '_'
nb.po:4425: msgstr lacks the keyboard accelerator mark '_'
nb.po:4663: msgstr lacks the keyboard accelerator mark '_'
nb.po:5427: msgstr lacks the keyboard accelerator mark '_'
nb.po:5432: msgstr lacks the keyboard accelerator mark '_'
nb.po:6842: msgstr lacks the keyboard accelerator mark '_'
nb.po:8730: msgstr lacks the keyboard accelerator mark '_'
nb.po:8820: msgstr lacks the keyboard accelerator mark '_'
nb.po:13016: msgstr lacks the keyboard accelerator mark '_'
nb.po:14859: msgstr lacks the keyboard accelerator mark '_'
nb.po:14934: msgstr lacks the keyboard accelerator mark '_'
nb.po:15011: msgstr lacks the keyboard accelerator mark '_'
nb.po:16629: msgstr lacks the keyboard accelerator mark '_'
nb.po:16815: msgstr lacks the keyboard accelerator mark '_'
nb.po:17014: msgstr lacks the keyboard accelerator mark '_'
nb.po:17029: msgstr lacks the keyboard accelerator mark '_'
nb.po:18025: msgstr lacks the keyboard accelerator mark '_'
nb.po:18029: msgstr lacks the keyboard accelerator mark '_'
nb.po:18155: msgstr lacks the keyboard accelerator mark '_'
nb.po:19095: msgstr lacks the keyboard accelerator mark '_'
nb.po:19260: msgstr lacks the keyboard accelerator mark '_'
nb.po:19277: msgstr lacks the keyboard accelerator mark '_'
nb.po:21007: format specifications in 'msgid_plural' and 'msgstr[0]' for argument 1 are not the same
msgfmt: found 30 fatal errors
2837 translated messages, 1806 fuzzy translations, 902 untranslated messages.

Reply:
I added some that were missing, but there are many more.